src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        StringBuilder results = new StringBuilder();
+        String importantSymbol = ",";
+        int operationTypeIndex = 0;
```

Comment:
The variable `importantSymbol` is essentially a delimiter for the CSV format. It would be more appropriate to name it `delimiter` or `csvDelimiter` to express its purpose more clearly.